src/components/queue/QueueList/QueueList.tsx

```diff
@@ -24,13 +27,28 @@ export interface QueueListProps {
  */
 const QueueList: FC<QueueListProps> = ({queue, showCompletedTickets}) => {
     const {currentUser, isTA} = useAuth();
-    const [tickets, ticketsLoading] = useTickets(queue.id, showCompletedTickets);
+    const [tickets, ticketsLoading] = useTickets(queue.id, false);
+    const shownTickets = tickets?.filter(t => !showCompletedTickets || t.status !== TicketStatus.StatusComplete);
     const [createTicketDialog, setCreateTicketDialog] = useState(false);
 
-    const inQueue = tickets && tickets.filter(ticket => ticket.user.Email == currentUser?.email).length > 0;
+    const [currentTime, setCurrentTime] = useState(new Date());
+
+    useEffect(() => {
+        const id = setInterval(() => {
+            setCurrentTime(new Date());
+        }, 5000);
+        return () => clearInterval(id);
+    });
+
```

Comment:
Use effect loop updates the time every 5 seconds (animations still run smoothly since it is 5) which update the state of the time.

Reply:
This likely does not need to be a useState for the currentTime and could just be made a variable.